chargeamps/external.py

```diff
@@ -153,3 +154,15 @@ async def set_chargepoint_connector_settings(
         connector_id = settings.connector_id
         request_uri = f"/api/{API_VERSION}/chargepoints/{charge_point_id}/connectors/{connector_id}/settings"
         await self._put(request_uri, json=payload)
+
+    async def remote_start(self, charge_point_id: str, connector_id: int, startAuth: StartAuth) -> None:
+        """Remote start chargepoint"""
+        payload = startAuth.to_dict()
+        request_uri = f"/api/{API_VERSION}/chargepoints/{charge_point_id}/connectors/{connector_id}/remotestart"
+        await self._put(request_uri, json=payload)
+
+    async def remote_stop(self, charge_point_id: str, connector_id: int, startAuth: StartAuth) -> None:
```

Comment:
Do you really need startAuth on stop?

Reply:
https://eapi.charge.space/swagger/index.html#/ChargePoint/ChargePoint_RemoteStop does say anything about this.

Reply:
You are correct! I'll remove it.